openedx/core/djangoapps/user_authn/api/tests/test_serializers.py

```diff
@@ -42,3 +45,38 @@ def test_mfe_context_serializer_default_response(self):
             serialized_data,
             default_expected_mfe_context_data
         )
+
+
+class TestMFEContextWelcomePageSerializer(TestCase):
+    """
+    Tests for MFEContextWelcomePageSerializer
+    """
+
+    def test_default_response(self):
+        """
+        Test default response for MFEContextWelcomePageSerializer
+        """
+
+        expected_response = {
+            'fields': None,
+            'extended_profile': [],
+        }
```

Comment:
I would suggest obtaining `expected_response` from the `constants` or the `test_data.py` file to ensure consistency.

Reply:
I don't think such a small response warrants adding it to mock_data file. The purpose of that file is to add constants so that the test file itself is legible and this is a very small response.